.eslintrc.js

```diff
@@ -30,13 +55,13 @@ module.exports = {
   extends: [
     'plugin:prettier/recommended',
     'plugin:react/recommended',
-    'prettier/@typescript-eslint',
-    'plugin:@typescript-eslint/recommended'
+    'prettier'
   ],
   parser: 'babel-eslint',
   plugins: ['standard', 'rulesdir'],
   rules: {
-    'no-unused-vars': 'error',
+    'no-unused-vars': 0,
```

Comment:
I think we can remove this line.

Reply:
`no-unused-vars` has issues when figuring out lexical scoped variables in `.ts` files. we rely on https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md

Reply:
```
{
  // note you must disable the base rule as it can report incorrect errors
  "no-unused-vars": "off",
  "@typescript-eslint/no-unused-vars": ["error"]
}
```